massgov/openmass
# DP-22535: Make paragraph for Events on org page

**Description:**
Added an Organization Events paragraph and refactored logic to display on an org_page node.


**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22535


**To Test:**
- Login and go to /node/add/org_page (or edit an existing org_page).
- Click on the "Content" tab.
- Expand the "Section Content" field.
- [ ] Verify there is an "Add Organization Events" button.
- Click the "Add Organization Events" button.
- [ ] Verify there is an "Event quantity" field.
- Add content to the fields and save (recommend editing an existing org_page with Event content).
- [ ] View the org_page and verify the content displays as expected.


**Screenshots/GIFs:**
![screencapture-mass-local-orgs-opioid-recovery-and-remediation-fund-advisory-council-2021-07-16-23_58_06](https://user-images.githubusercontent.com/1434979/126025006-f7f3d700-837b-440c-b40c-880facb9aa82.png)

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
